### DISCUSSION TITLE
求视频里提到的openai的小哥写的关于大规模分布式训练的博客的网址

### DISCUSSION THREAD

**Discussion by lyxichigoichie** [mli/paper-reading#263] [Q&A]
之前视频里面沐神还是毅神提到的一篇openai的小哥写的关于大规模分布式训练的博客有同学还记得网址吗。我当时没记下来，现在找不到在哪了

**Comment by liu33333** [comment]
有一个讲大模型训练的blog https://lilianweng.github.io/posts/2021-09-25-train-large/ 但不是小哥😂

**Reply by lyxichigoichie** [reply]
非常感谢🙏
